Sources/Apollo/ApolloClientProtocol.swift

```diff
@@ -37,10 +37,12 @@ public protocol ApolloClientProtocol: class {
   /// - Parameters:
   ///   - query: The query to fetch.
   ///   - cachePolicy: A cache policy that specifies when results should be fetched from the server or from the local cache.
+  ///   - callbackQueue: A dispatch queue on which the result handler will be called. Defaults to the main queue.
   ///   - resultHandler: [optional] A closure that is called when query results are available or when an error occurs.
   /// - Returns: A query watcher object that can be used to control the watching behavior.
   func watch<Query: GraphQLQuery>(query: Query,
                                   cachePolicy: CachePolicy,
+                                  callbackQueue: DispatchQueue,
```

Comment:
This is a breaking change so I'm not sure what the standard procedure is for this.

We don't use this method in our app and create the watcher directly, but I think this change aligns with all the other network methods in the protocol

Reply:
Luckily we're still at 0.x so you can sneak this one in. 😇

You should update the docs to `should default to the main queue` since you can't declare a default on a protocol.